app/templates/dev-flush-page.html

```diff
@@ -0,0 +1,118 @@
+{% set _theme="default" %}
+{% extends 'layouts/_base.html' %} {% block page_title %}EQ Development Flush Data{% endblock %} {% block content %}
+
+<!-- this page needs a total rethink, but this tidies things up a little for now -->
```

Comment:
The flush process needs just 4 parameters to work, I didn't want the overhead of another 15 variables being passed around. I am all for reuse but this gave me a clean end point without the need for logic in the post. 

As this will be used stand alone, I wanted to make the process as realistic a possible,  respondent home will only pass the 4 parameters to make it work, therefore the dev_flush tool should do the same